website/docs/faqs/Accounts/delete-users.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How do I delete a user in dbt Cloud?
+description: "Deleting a user in dbt Cloud"
+sidebar_label: 'How to delete a user'
+id: delete-users
+
+---
+
+This article outlines the steps for deleting a user in dbt Cloud. If the user has a 'developer' license-type, this will open up their seat for another user or allow the admins to lower the total number of billable seats. You must be an account owner/admin to perform these actions.
```

Comment:
```suggestion
This article outlines the steps for deleting a user in dbt Cloud. If the user has a 'developer' license type, this will open up their seat for another user or allow the admins to lower the total number of seats. You must be an account owner or admin to perform these actions.
```